src/main/java/org/sagebionetworks/bridge/spring/controllers/CRCController.java

```diff
@@ -0,0 +1,372 @@
+import org.sagebionetworks.bridge.exceptions.EntityNotFoundException;
+import org.sagebionetworks.bridge.exceptions.NotAuthenticatedException;
+import org.sagebionetworks.bridge.models.DateRangeResourceList;
+import org.sagebionetworks.bridge.models.StatusMessage;
+import org.sagebionetworks.bridge.models.accounts.Account;
+import org.sagebionetworks.bridge.models.accounts.AccountId;
+import org.sagebionetworks.bridge.models.accounts.SignIn;
+import org.sagebionetworks.bridge.models.accounts.StudyParticipant;
+import org.sagebionetworks.bridge.models.accounts.UserSession;
+import org.sagebionetworks.bridge.models.apps.App;
+import org.sagebionetworks.bridge.models.reports.ReportData;
+import org.sagebionetworks.bridge.services.ParticipantService;
+import org.sagebionetworks.bridge.services.ReportService;
+
+import ca.uhn.fhir.context.FhirContext;
+import ca.uhn.fhir.parser.IParser;
+
+@CrossOrigin
+@RestController
+public class CRCController extends BaseController {
+    
+    static final String TIMESTAMP_FIELD = "state_change_timestamp";
+    static final String USER_ID_VALUE_NS = "https://ws.sagebridge.org/#userId";
+    static final String APP_ID = "czi-coronavirus";
+    static final String OBSERVATION_REPORT = "observation";
+    static final String PROCEDURE_REPORT = "procedurerequest";
+    static final String APPOINTMENT_REPORT = "appointment";
+    static final String USERNAME = "A5hfO-tdLP_eEjx9vf2orSd5";
+    static final String TEST_USERNAME = "pFLaYky-7ToEH7MB6ZhzqpKe";
+    // This is thread-safe and it's recommended to reuse an instance because it's expensive to create;
+    static final FhirContext FHIR_CONTEXT = FhirContext.forDstu3();
+    static final LocalDate JAN1 = LocalDate.parse("1970-01-01");
+    static final LocalDate JAN2 = LocalDate.parse("1970-01-02");
+    static final Map<String, String> ACCOUNTS = ImmutableMap.of(
+            USERNAME, APP_ID, 
+            TEST_USERNAME, API_APP_ID);
+    
+    static enum AccountStates {
+        ENROLLED,
+        SELECTED, 
+        DECLINED,
+        TESTS_REQUESTED,
+        TESTS_SCHEDULED,
+        TESTS_COLLECTED,
+        TESTS_AVAILABLE
+    }
+    
+    private static final Set<String> ALL_STATES = Arrays.stream(AccountStates.values())
+            .map(e -> e.name().toLowerCase())
+            .collect(toImmutableSet());
+    
+    private ParticipantService participantService;
+    
+    private ReportService reportService;
+    
+    @Autowired
+    final void setParticipantService(ParticipantService participantService) {
+        this.participantService = participantService;
+    }
+    
+    @Autowired
+    final void setReportService(ReportService reportService) {
+        this.reportService = reportService;
+    }
+    
+    DateTime getTimestamp() {
+        return DateTime.now().withZone(DateTimeZone.UTC);
+    }
+    
+    @PostMapping("/v1/cuimc/participants/{userId}")
+    public StatusMessage updateParticipant(@PathVariable String userId) {
```

Comment:
This method takes in the userId but never uses it.

Reply:
This is definitely a bug. Will also add a test of this method in the integration tests (and fixed it here).